server/reportGenerator/src/reportGen.ts

```diff
@@ -141,12 +142,10 @@ export const generateReport = async (event: SQSEvent, context: AWSLambda.Context
 	// need to await, otherwise if the lambda function terminates before the upload is finished, it won't complete
 	await s3.upload(s3params).promise();
 
-	// Add batch to file zip queue
+	// Add batch to email queue
 	const params: SendMessageRequest = {
-		MessageBody: '{ "queryStringParameters": { "id": "' + String(key) + '" } }', // zipfile accesses event["queryStringParameters"]["id"]
-		QueueUrl: process.env.FILE_ZIP_QUEUE as string,
+		MessageBody: '{ "jobKey": "' + String(key) + '" }',
+		QueueUrl: process.env.EMAIL_QUEUE as string,
 	};
 	pushToQueue(params);
```

Comment:
so does this now call add to the send email queue? Do I need to change anything in my send email code to get the url for the report?

Reply:
The url for a report is https://afternoon-woodland-24079.herokuapp.com/batchreportpage/JOB_KEY?pwd=PASSWORD 
JOB_KEY is the job key and PASSWORD is eventually stored in the job table as { id: JOB_KEY { password: PASSWORD}}.
Nothing generates the password yet (i think), but since the email is the first time the password is used, it's probably best to generate and store the password in send email